public/app/plugins/datasource/prometheus/components/PromQueryEditor.tsx

```diff
@@ -52,6 +68,8 @@ export class PromQueryEditor extends PureComponent<PromQueryEditorProps, State>
       formatOption: FORMAT_OPTIONS.find((option) => option.value === query.format) || FORMAT_OPTIONS[0],
       intervalFactorOption:
         INTERVAL_FACTOR_OPTIONS.find((option) => option.value === query.intervalFactor) || INTERVAL_FACTOR_OPTIONS[0],
+      // Step option
+      stepOption: STEP_OPTIONS.find((option) => option.value === query.stepOption) || STEP_OPTIONS[0],
```

Comment:
`STEP_OPTIONS[0]` should be extracted as `DEFAULT_STEP_OPTION`.

Reply:
Don't know if i understood correctly but i changed it to `DEFAULT_STEP_OPTION`